chromadb/cli/cli.py

```diff
@@ -131,6 +131,7 @@ def vacuum(
     settings.is_persistent = True
     settings.persist_directory = path
     system = System(settings=settings)
+    system.start()
```

Comment:
fixes a bug where if `trigger_vector_segments_max_seq_id_migration()` below attempted to load a segment, it would throw because the SQLite component wasn't technically started
this scenario happens when upgrading from an old version or when any collection had not yet hit its first sync_threshold persist trigger
was not caught because the CLI test didn't add any collections, I updated the test to trigger this path